### PR TITLE
Materials datums now apply to all items with materials.

### PR DESCRIPTION
**Pull request by ghost**
:cl: Qustinnus
 add: material datums now apply to all items with materials in them
/:cl:

this makes mining somewhat more dangerous as plasma is now something that can explode and uranium is radioactive. buffed miner rad protection as a counter measure.

### CODE REVIEW

**Comment by Dawson1917** [comment]
Miners getting radiation poisoning from mining is terrible, and buffing their rad resistance won't help much at all. You can easily have over >30 uranium ores in two minutes of mining with cutters, and I do not want to sprint back to the station to hand over uranium before it kills me, nor do I want to randomly get mutated or radiation stuns while fighting megafauna because I walked over a green ore 5 minutes ago and never noticed.

Uranium ore shouldn't irradiate.

**Comment by Qustinnus** [comment]
its not gonna hurt you in the bag.

**Comment by Dawson1917** [comment]
Fair enough. It might be good to spawn uranium ore on the floor and see how quickly you reach >100% radiation with & without mining gear. 90% rad resist on the miner hardsuit (which are purchasable by anyone now) is the same as the Chief Engineer's hardsuit, and 80% on the explorer suit is above even the engineering hardsuit. Too much, and if it's necessitated by uranium ore irradiating miners too quickly then something will have to be done about that rather than give miners more boosted stats

**Comment by Qustinnus** [comment]
i forgot about public mining, I'll go test it.

**Comment by cacogen** [comment]
i really like the idea in theory i just wonder what will have to change for it to work in practice

**Comment by Qustinnus** [comment]
this was gonna happen at some point anyways.

**Comment by CRITAWAKETS** [comment]
Uranium ore should not make like, any radiation besides like 4 per tick for a full stack of it. Reminder that almost all of the uranium in uranium ore is barely radioactive Uranium-238. The real radioactive cool beans is mainly Uranium-235. I would imagine the ORM processes the uranium.

**Comment by kopoba** [comment]
Please add more fire resistance to miner's jumpsuit its burn down in one step in lava.

**Comment by GuyonBroadway** [comment]
Iirc unrefined uranium isn't radioactive enough to pose a threat to anyone wearing basic protection.

**Comment by nemvar** [comment]
irl uranium also isn't green but here we are

**Comment by PKPenguin321** [comment]
It's space uranium

**Comment by GuyonBroadway** [comment]
In any case, giving miners double jeopardy just for doing thier jobs is kinda shitty. They already got mobs and ash storms to contend with, giving them radiation poisoning and plasma fires to contend with unpreventably is pretty bad.

**Comment by Qustinnus** [comment]
okay but you realize that it wont do shit unless you fire a laser beam (plasma) at it or dont pick it up in your mining bag / ore box right (uranium)

**Comment by Dawson1917** [comment]
> 
> 
> okay but you realize that it wont do shit unless you fire a laser beam (plasma) at it or dont pick it up in your mining bag / ore box right (uranium)

plasma cutter shots and hits ignite plasma if it hits the sprite, and

ash drake has a habit of breaking open rocks or igniting any plasma you didn't collect while digging out an arena

**Comment by Qustinnus** [comment]
I suppose thats true but I honestly think thats an acceptable mechanic. If you leave a volatile material out in the open and it goes off due to things in the environment, that sounds like a cool situation. And unlike other things it's not necesarily a hardcore chore to just pickup the plasma you mine.

**Comment by Qustinnus** [comment]
Thought of a solution, a new var called mat_effectiveness which nerfs the amount actually used during material application, this'll also prevent coins from being used as 1 tile nukes.

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/modules/mining/ores_coins.dm [hunk 1]
```diff
@@ -17,6 +17,7 @@
 	var/points = 0 //How many points this ore gets you from the ore redemption machine
 	var/refined_type = null //What this ore defaults to being refined into
 	novariants = TRUE // Ore stacks handle their icon updates themselves to keep the illusion that there's more going
+	material_modifier = 0.5 //Less effective as its not refined.
```
Comment (81Denton) [inline code/modules/mining/ores_coins.dm]: Make sure to test that this is low enough so that uranium ore is below `#define RAD_MINIMUM_CONTAMINATION 350`

**Comment by Qustinnus** [comment]
deprecated by #46525

File: code/modules/mining/ores_coins.dm (hunk 1 above)
Comment (Qustinnus) [reply code/modules/mining/ores_coins.dm]: I solved this in my new PR by making the bags rad insulating.

**Pull request closed without merging**